feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/BookmarkIcon.kt

```diff
@@ -25,27 +25,26 @@ import io.github.droidkaigi.confsched2023.feature.sessions.R.raw
 @Composable
 fun BookmarkIcon(
     contentDescription: String,
-    onBookmarkClickStatus: Boolean?,
+    onBookmarkClickStatus: Boolean,
     onReachAnimationEnd: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
     val lottieComposition by rememberLottieComposition(RawRes(raw.add_to_bookmark_lottie))
-    val animationRange = 0.1f..1f
     var isPlaying by remember { mutableStateOf(false) }
 
-    val progress by animateLottieCompositionAsState(
+    val state = animateLottieCompositionAsState(
```

Comment:
What if we name this `lottieCompositionState`?

Reply:
it's more understandable.
i changed it from `state` to `lottieState`
(https://github.com/DroidKaigi/conference-app-2023/pull/1131/commits/5e7f7cf99450149105ee10aadd5a4773fce37cb1)